DINA-Web-API-Guidelines.md

```diff
@@ -558,19 +558,14 @@ Look at examples of RESTful apis for image thumbnails, & other image manipulatio
 Documentation
 -------------
 
--   Each DINA compliant Web REST API **MUST** blueprint https://apiblueprint.org/ for their API.
-
-
-**[under-discussion]** replace the below with a reference to a apiray-blueprint file (i.e media.apib)
-Here is the above JSON in the POST body formatted a little better (this is just an example of what we might want in the REQUEST JSON):
-
+-   Each DINA compliant Web REST API **MUST** have an agreed OpenAPI Specification Version 3 for their API under source control in https://github.com/DINA-Web.
+-   Specification repositories should be able to automaticaly generate API documentation from the OpenAPI Specification file.
```

Comment:
I'd rather say that the APIs themselves should be "self-documented" by a specific endpoint that returns the OpenAPI specification file for the respective module. This would allow for actionable documentation (e.g. with SwaggerUI).

Reply:
But I agree, that the the specs-repos should (or even must?) contain a static OpenAPI specification file.

Reply:
The original sentence was referring to the repositories on DINA-Web, the master copy of the API specification. The idea is to avoid different implementations to have a different specifications. So the master copy is stored under source-control, in a neutral/community space (DINA-Web/...-specs).

Reply:
OK, I see. Sorry for the confusion.